client-common/src/main/java/org/apache/cassandra/sidecar/common/response/GossipInfoResponse.java

```diff
@@ -275,7 +282,8 @@ protected enum GossipField
          * as a comma-separated list.
          **/
         SSTABLE_VERSIONS,
-        DISK_USAGE;
+        DISK_USAGE,
+        INDEX_STATUS;
```

Comment:
Add a comment to indicate since what Cassandra verison the gossip value is available.
And move `;` to a new line, to avoid showing the diff on the existing enum in the future. 

I can also update it on merging, if you agree with the suggestion, but do not feel like to push a new commit. 

```suggestion
        INDEX_STATUS, // Intorduced in Cassandra 5.0 for SAI 
        ;
```

Reply:
Fixed typo and updated 👍